packages/ag-charts-enterprise/src/features/zoom/zoomContextMenu.ts

```diff
@@ -66,11 +64,11 @@ export class ZoomContextMenu {
         }
     }
 
-    private onZoomToHere({ event }: ContextMenuActionParams, props: ZoomProperties) {
+    private onZoomToHere({ event }: AgNodeContextMenuActionEvent, props: ZoomProperties) {
         const { rect } = this;
         const { enabled, isScalingX, isScalingY, minRatioX, minRatioY } = props;
 
-        if (!enabled || !rect || !event || !event.target) return;
+        if (!enabled || !rect || !event || !event.target || !(event instanceof MouseEvent)) return;
```

Comment:
Nit: Why do we need to check for `MouseEvent` here and below now?

Reply:
It is needed because `ContextMenuActionParams['event']` is of type `MouseEvent`, whereas `AgNodeContextMenuActionEvent['event']` is of type `Event`.

https://github.com/ag-grid/ag-charts/blob/1a7fe5c9c9e1af099fddf2a33d5215e3b51b98f3/packages/ag-charts-community/src/chart/interaction/contextMenuRegistry.ts#L36

https://github.com/ag-grid/ag-charts/blob/1a7fe5c9c9e1af099fddf2a33d5215e3b51b98f3/packages/ag-charts-community/src/options/chart/eventOptions.ts#L3

Therefore we need to check that the event is a `MouseEvent` so that we can read the `clientX, clientY` properties a few lines below.

We could (should?) perhaps change the `AgChartEvent` interface so that we can specify an event subtype:
```
interface AgChartEvent<T extends string, E extends Event> {
    type: T;
    event: E;
}
```